pkg/kn/commands/broker/delete_test.go

```diff
@@ -36,7 +36,7 @@ func TestBrokerDelete(t *testing.T) {
 
 	out, err := executeBrokerCommand(eventingClient, "delete", brokerName)
 	assert.NilError(t, err, "Broker should be deleted")
-	util.ContainsAll(out, "Broker", brokerName, "deleted", "namespace", "default")
+	assert.Assert(t, util.ContainsAll(out, "Broker", brokerName, "deleted", "namespace", "default"))
```

Comment:
hmm, nice catches... it’s easy read this as `util.ContainAll(...)` doing the checks... but it’s not. Well done.

Reply:
Yeah, ran into this when writing some tests for earlier pull requests. Maybe original thought process was that assertion was taking place in ContainsAll(...) as you said.